diff --git a/dist/node/chunks/dep-557f29e6.js b/dist/node/chunks/dep-557f29e6.js
index 0ef6357f94a83d600c09149205017220e0807ca7..ba983e936c857a06980fb9a9c133938d9842395e 100644
--- a/dist/node/chunks/dep-557f29e6.js
+++ b/dist/node/chunks/dep-557f29e6.js
@@ -34617,7 +34617,7 @@ function tryNodeResolve(id, importer, options, targetWeb, depsOptimizer, ssr, ex
         let resolvedId = id;
         if (isDeepImport) {
             if (!pkg?.data.exports && path$n.extname(id) !== resolvedExt) {
-                resolvedId += resolvedExt;
+                resolvedId = resolved.id.slice(resolved.id.indexOf(id));
             }
         }
         return { ...resolved, id: resolvedId, external: true };